Makefile

```diff
@@ -38,12 +38,30 @@ lint: lint.sub
 push: push.sub
 test: test.sub
 
+clean:
```

Comment:
we need to reconcile this with line 35. I assume the ideal solution is delete line 35 and change line 41 to:

```clean: clean.sub
        rm -f .dinit
        rm -f .buildImage
        ...
```